### PR TITLE
Add LICENSE

### PR DESCRIPTION
**Pull request by dunn**
cc @mikemcquaid @domt4 @edavis

### CODE REVIEW

**Comment by rwhogg** [comment]
Thanks a bunch, I was really hoping you would do this :smile:

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Copyright 2014-2016 Eric Davis and Homebrew contributors.
```
Comment (mikemcquaid) [inline LICENSE]: @edavis For simplicity would you mind just being included under `Homebrew contributors` here?

**Comment by edavis** [comment]
I, Eric Davis, release my contributions under the BSD 2-Clause License.

cc @dunn

File: LICENSE (hunk 1 above)
Comment (edavis) [inline LICENSE]: @mikemcquaid Nope, I have no issue with that.

**Comment by dunn** [comment]
Thanks, everyone!!

**Pull request closed without merging**